ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/router-traffic-engineering.j2

```diff
@@ -0,0 +1,43 @@
+{% if router_traffic_engineering is arista.avd.defined %}
+
+## Router Traffic-Engineering
+{%     if router_traffic_engineering.segment_routing is arista.avd.defined %}
+
+### Segment Routing Summary
+
+- SRTE is enabled.
+{%         if router_traffic_engineering.segment_routing.colored_tunnel_rib is arista.avd.defined(true) %}
+
+- system-colored-tunnel-rib is enabled
+{%         endif %}
+{%         if router_traffic_engineering.segment_routing.policy_endpoints is arista.avd.defined %}
+
+#### SRTE Policies
+
+| Endpoint | Color | Preference | Name | Description | Label Stack | Index  | Weight | Explicit Null |
+| -------- | ----- | ---------- | ---- | ----------- | ----------- | ------ | ------ | ------------- |
+{%             for endpoint in router_traffic_engineering.segment_routing.policy_endpoints | arista.avd.natural_sort('address') %}
+{%                 for color in endpoint.colors | arista.avd.natural_sort('value') %}
+{%                     set name = color.name | arista.avd.default('-') %}
+{%                     set description = color.description | arista.avd.default('-') %}
+{%                     for pathgroup in color.path_group | arista.avd.natural_sort('preference') %}
+{%                         set pathgroup_preference = pathgroup.preference | arista.avd.default('-') %}
+{%                         set expnull = pathgroup.explicit_null | arista.avd.default('-') %}
+{%                         for labelstack in pathgroup.segment_list | arista.avd.natural_sort('index') %}
```

Comment:
The sort key should be ```label_stack```.

Reply:
From what I understood the index parameter is an integer for "indexing" the list of segment-lists, which looks like a better candidate than segment-paths as the preferred sort_key. That being said users may or may not specify, which means it can fall back to segment-list (among others).

Reply:
I can switch it back to label_stack if that's more desirable as a sort_key.

Reply:
Yeah, but in eos_cli_config_gen we try to replicate same output of eos. So we are sorting based on label_stack.